### DISCUSSION TITLE
Terraform Provider and Ansible Module

### DISCUSSION THREAD

**Discussion by natesales** [fosshostorg/aarch64#11] [Ideas]
It would be neat to have a Terraform Provider and Ansible Module for aarch64.com, based on the Go and Python API clients ([aarch64-client-go](https://github.com/natesales/aarch64-client-go) and [aarch64-client-python](https://github.com/natesales/aarch64-client-python)).

**Comment by lanefu** [comment]
excited about this....

I'd say go ahead and aim for an aarch64.com ansible collection.      given you have the python client already.. most time spent will be boiler-plate cleanup documentation etc for the module.
